grails-app/controllers/au/org/ala/userdetails/RegistrationController.groovy

```diff
@@ -165,6 +177,10 @@ class RegistrationController {
         }
     }
 
+    def quoteValue(String value) {
```

Comment:
`quoteValue` should be optional switched via configuration like in ALA CAS
There should also be a configurable `encode` on the cookie value as well (defaults to false as ALA CAS does)